ramlfications/utils.py

```diff
@@ -117,34 +135,23 @@ def save_data(output_file, mime_types):
     with open(output_file, "w") as f:
         json.dump(mime_types, f)
 
+def setup_logger(message):
```

Comment:
I think it'd be better to rename `message` as something like `logger_name`, and maybe just use the function name as the logger name.